tests/pages/productDetails.test.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import ProductDetails, { getServerSideProps } from '../../src/pages/productDetails';
+import { ProductInfo } from '../../src/interfaces';
+import { getMockContext } from '../testData/mockData';
+
+const mockproductDetails: ProductInfo = {
+    productPrice: '',
+    productName: '',
+    productNameError: '',
+    productPriceError: '',
+};
+
+describe('pages', () => {
+    describe('productDetails', () => {
+        it('should render correctly', () => {
+            const tree = shallow(
+                <ProductDetails product={mockproductDetails} operator="bus company" hintText="Test Zone" />,
+            );
+            expect(tree).toMatchSnapshot();
+        });
+    });
+
+    describe('productDetailsGetServerSideProps', () => {
```

Comment:
I would keep this as two separate words 'productDetails getServerSideProps' or nest a 'getServerSideProps' describe within the 'productDetails' describe

Reply:
Separated this and nested within the productDetails describe